### PR TITLE
ErrorResponse: add additional required properties from API Guild Erro…

### PR DESCRIPTION
**Pull request by kenmccracken**
…r Message spec.

### CODE REVIEW

File: here-oauth-client/src/test/java/com/here/account/oauth2/ErrorResponseTest.java [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+        String actualCause = errorResponse.getCause();
+        assertTrue("expected cause " + null + ", actual " + actualCause,
+                null == (actualCause));
+
+        String actualAction = errorResponse.getAction();
+        assertTrue("expected action " + null + ", actual " + actualAction,
+                null == (actualAction));
+
+        String actualCorrelationId = errorResponse.getCorrelationId();
+        assertTrue("expected correlationId " + correlationId + ", actual " + actualCorrelationId,
```
Comment (srrajago) [inline here-oauth-client/src/test/java/com/here/account/oauth2/ErrorResponseTest.java]: should this be "null"?

File: here-oauth-client/src/test/java/com/here/account/oauth2/ErrorResponseTest.java [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+                errorCode.equals(actualErrorCode));
+
+        String actualMessage = errorResponse.getMessage();
+        assertTrue("expected message " + message + ", actual " + actualMessage,
+                message.equals(actualMessage));
+
+    }
+
+    @Test
+    public void test_correlationId() {
```
Comment (srrajago) [inline here-oauth-client/src/test/java/com/here/account/oauth2/ErrorResponseTest.java]: why is this test named test_correlationId? Isn't it checking the presence of all the 6 new properties

File: here-oauth-client/src/test/java/com/here/account/oauth2/ErrorResponseTest.java (hunk 1 above)
Comment (srrajago) [inline here-oauth-client/src/test/java/com/here/account/oauth2/ErrorResponseTest.java]: The expected correlationId should be null.

Comment (kenmccracken) [reply here-oauth-client/src/test/java/com/here/account/oauth2/ErrorResponseTest.java]: Thanks, I have pushed changes to fix this.

File: here-oauth-client/src/test/java/com/here/account/oauth2/ErrorResponseTest.java (hunk 2 above)
Comment (kenmccracken) [reply here-oauth-client/src/test/java/com/here/account/oauth2/ErrorResponseTest.java]: Thanks, I have pushed changes to fix this.

**Pull request merged**